### ISSUE TITLE
Recover a PoV & PeristedValidationData

### ISSUE THREAD

**Issue by pepyakin** [paritytech/cumulus#1423]
Several times I felt the need in getting a PoV for a given parablock. Specifically, for debugging or benchmarking. I think it would be great if cumulus allowed me to recreate the collation for a given parachain block.

Assuming I had an archive parachain node that runs with an archive relay node, I should theoretically be able to recreate a PoV for the given block, given that all the required data is supplied (specifically, `PersistedValidationData` and/or relay-parent).

Alternatively, a service that crawls the relay chain and recovers the PoV & PersistedValidationData would work as well.

**Comment by bkchr** [comment]
Should not be that complicated, the only missing piece would be the relay parent. The relay parent is only in the runtime and not on the client side. It would be solvable, but would maybe require a new runtime api. 

Would you like to have it as CLI or RPC?

**Comment by crystalin** [comment]
I agree it would be a nice feature. Also through CLI is enough for us, something like:
`./parachain pov extract <hash or number>`
